src/Turn.js

```diff
@@ -0,0 +1,20 @@
+const Card = require('../src/Card');
+
+class Turn {
+  constructor(guess, card) {
```

Comment:
Need to removed these from the scope of the constructor, also add properties to instantiate to.